### DISCUSSION TITLE
ParadeDB as the main database

### DISCUSSION THREAD

**Discussion by olivermt** [ubicloud/ubicloud#2156] [Q&A] [answered]
The console is a bit thin on information on what it MEANS to use ParadeDB.
One of the benefits of ParadeDB is that you get everything you need in one db.

It says the DB is managed by paradedb, does that mean that the Ubicloud team is not involved?
When configuring it in the console UI, the options seem identical to the main postgres offering.

Edit:

I now saw:
> Our managed ParadeDB service combines the unique expertise of both Ubicloud and ParadeDB teams. Underlying infrastructure is managed by the Ubicloud team and the ParadeDB instances are managed by the ParadeDB team. This allows you to access both teams’ technical expertise as needed.

Still curious about whether or not Ubicloud as postgresql experts recommend to use ParadeDB as the main DB or not.

**Comment by ozgune** [comment]
Hi @olivermt - Thank you for getting in touch!

I think the answer to your question depends on your use-case. ParadeDB is an Elasticsearch alternative built on Postgres; and provides powerful full-text search and analytics capabilities. https://www.ubicloud.com/docs/managed-postgresql/paradedb-on-ubicloud

If your use-case aligns with the above description, we'd recommend that you use ParadeDB. If you need a stock PostgreSQL instead, then the default managed database would also work for you.

Does this help answer your question? If not, please let us know and we'd be happy to provide more information.

**Reply by byucesoy** [reply, marked as answer]
Hi @olivermt, primary architect of Ubicloud's managed Postgres services is here :wave:. I wanted to add few details to Ozgun's answer.

Both regular Ubicloud PostgreSQL and ParadeDB PostgreSQL runs on the same infrastructure. The infrastructure is operated and managed by the Ubicloud team. For example in both cases, Ubicloud team is responsible taking backups, doing security upgrades and ensuring that the server is healthy overall.

In ParadeDB PostgreSQL case, there are few additional benefits;
- It comes with the extensions developed by ParadeDB team; `pg_search` and `pg_analytics`, which significantly increases PostgreSQL's full text search capabilities.
- You also get access to ParadeDB team's expertise. We are close partners with them and they would be aware that you are running a ParadeDB database backed by Ubicloud. They can provide ParadeDB specific help and tips when needed.
- As you mentioned, you can keep one database for both your transactional and analytical needs.

**Reply by olivermt** [reply]
That last part was exactly what I was wondering. Essentially "can I trust ParadeDB with my critical core data as much as I can the regular Ubicloud postgres or is it meant to be ran in addition with the elasticsearch features" :) 

That answers my question I think, and we'll be going with ParadeDB instances instead of the regular postgres.
